src/main/java/hw3/ex1/HomePage.java

```diff
@@ -0,0 +1,155 @@
+package hw3.ex1;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import utils.AbstractBasePage;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HomePage extends AbstractBasePage {
+
+    private static final String URL = "https://epam.github.io/JDI/index.html";
+
+    @FindBy(id = "user-icon")
+    private WebElement userIcon;
+
+    @FindBy(id = "name")
+    private WebElement usernameTextField;
+
+    @FindBy(id = "password")
+    private WebElement passwordTextField;
+
+    @FindBy(id = "login-button")
+    private WebElement loginButton;
+
+    @FindBy(id = "user-name")
+    private WebElement usernameLabel;
+
+    @FindBy(xpath = "//ul[@class='uui-navigation nav navbar-nav m-l8']")
+    private WebElement header;
+
+    @FindBy(xpath = "//div[@class='row clerafix benefits']")
+    private WebElement benefits;
+
+    @FindBy(xpath = "//h3[@name='main-title']")
```

Comment:
Could be used DOM locator

Reply:
Solved: I change it on `@FindBy(name = "main-title")`